<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakEvenCalculator</class>
 <widget class="QWidget" name="BreakEvenCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forex Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>401</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>1</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Trade Details</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>33</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Entry Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>67</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Trade Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEntryPrice">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLotSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>65</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Lot Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>33</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Loss Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditStopLoss">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>31</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>66</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Take Profit Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTakeProfit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>65</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>107</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Cost and Fees</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>33</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Spread</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>33</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSpread">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>32</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Spread in pips</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Commission</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCommission">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>31</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Spread in pips</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSwap">
     <property name="geometry">
      <rect>
       <x>294</x>
       <y>31</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Spread in pips</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>182</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Risk Management</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Risk %</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>32</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Account Balance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAccountBalance">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Spread in pips</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxRisk">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>31</y>
       <width>62</width>
       <height>26</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Percentage</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonCalculate">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>261</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>294</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Risk-Reward Ratio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRiskRewardRatio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>294</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>297</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Total Cost</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTotalCost">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>296</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBreakEvenPrice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>328</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>329</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Break Even Price</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="aboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="aboutForexCalculator">
   <property name="text">
    <string>About Forex Calculator</string>
   </property>
  </action>
  <action name="actionPip_calculator">
   <property name="text">
    <string>Pip Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionMargin_calculator">
   <property name="text">
    <string>Margin Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLot_Size_Calculator">
   <property name="text">
    <string>Lot Size Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionProfit_and_Loss_Calculator">
   <property name="text">
    <string>Profit and Loss Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPosition_Size_Calculator">
   <property name="text">
    <string>Position Size Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRisk_Reward_Calculator">
   <property name="text">
    <string>Risk/Reward Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSwap_Calculator">
   <property name="text">
    <string>Swap Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionCurrency_Converter">
   <property name="text">
    <string>Currency Converter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionFibonacci_Calculator">
   <property name="text">
    <string>Fibonacci Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionVolatility_Calculator">
   <property name="text">
    <string>Volatility Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionBreak_Even_Calculator">
   <property name="text">
    <string>Break-Even Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionCompound_Interest_Calculator">
   <property name="text">
    <string>Compound Interest Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionChinese_2">
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="text">
    <string>Arabic</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionPortuguese">
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="actionGerman">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
  <action name="actionHindi">
   <property name="text">
    <string>Hindi</string>
   </property>
  </action>
  <action name="actionBengali">
   <property name="text">
    <string>Bengali</string>
   </property>
  </action>
  <action name="actionPunjabi">
   <property name="text">
    <string>Punjabi</string>
   </property>
  </action>
  <action name="actionUrdu">
   <property name="text">
    <string>Urdu</string>
   </property>
  </action>
  <action name="actionVietnamese">
   <property name="text">
    <string>Vietnamese</string>
   </property>
  </action>
  <action name="actionIndonesian_Bahasa_Indonesia">
   <property name="text">
    <string>Indonesian (Bahasa Indonesia)</string>
   </property>
  </action>
  <action name="actionTagalog_Filipino">
   <property name="text">
    <string>Tagalog (Filipino)</string>
   </property>
  </action>
  <action name="actionKorean">
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="actionTamil">
   <property name="text">
    <string>Tamil</string>
   </property>
  </action>
  <action name="actionThai">
   <property name="text">
    <string>Thai</string>
   </property>
  </action>
  <action name="actionBurmese">
   <property name="text">
    <string>Burmese</string>
   </property>
  </action>
  <action name="actionKhmer">
   <property name="text">
    <string>Khmer</string>
   </property>
  </action>
  <action name="actionNepali">
   <property name="text">
    <string>Nepali</string>
   </property>
  </action>
  <action name="actionHindi_2">
   <property name="text">
    <string>Hindi</string>
   </property>
  </action>
  <action name="actionBengali_2">
   <property name="text">
    <string>Bengali</string>
   </property>
  </action>
  <action name="actionJapanese_2">
   <property name="text">
    <string>Japanese</string>
   </property>
  </action>
  <action name="actionKorean_2">
   <property name="text">
    <string>Korean</string>
   </property>
  </action>
  <action name="actionVietnamese_2">
   <property name="text">
    <string>Vietnamese</string>
   </property>
  </action>
  <action name="actionThai_2">
   <property name="text">
    <string>Thai</string>
   </property>
  </action>
  <action name="actionFrench_2">
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionGerman_2">
   <property name="text">
    <string>German</string>
   </property>
  </action>
  <action name="actionRussian_2">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionPortuguese_2">
   <property name="text">
    <string>Portuguese</string>
   </property>
  </action>
  <action name="actionItalian">
   <property name="text">
    <string>Italian</string>
   </property>
  </action>
  <action name="actionPersian_Farsi">
   <property name="text">
    <string>Persian (Farsi)</string>
   </property>
  </action>
  <action name="actionTurkish">
   <property name="text">
    <string>Turkish</string>
   </property>
  </action>
  <action name="actionHebrew">
   <property name="text">
    <string>Hebrew</string>
   </property>
  </action>
  <action name="actionPashto">
   <property name="text">
    <string>Pashto</string>
   </property>
  </action>
  <action name="actionSwahili">
   <property name="text">
    <string>Swahili</string>
   </property>
  </action>
  <action name="actionAmharic">
   <property name="text">
    <string>Amharic</string>
   </property>
  </action>
  <action name="actionHausa">
   <property name="text">
    <string>Hausa</string>
   </property>
  </action>
  <action name="actionYoruba">
   <property name="text">
    <string>Yoruba</string>
   </property>
  </action>
  <action name="actionZulu">
   <property name="text">
    <string>Zulu</string>
   </property>
  </action>
  <action name="actionHaitian_Creole">
   <property name="text">
    <string>Haitian Creole</string>
   </property>
  </action>
  <action name="actionQuechua">
   <property name="text">
    <string>Quechua</string>
   </property>
  </action>
  <action name="actionGuarani">
   <property name="text">
    <string>Guarani</string>
   </property>
  </action>
  <action name="actionNahuatl">
   <property name="text">
    <string>Nahuatl</string>
   </property>
  </action>
  <action name="actionRisk_Management_Calculator">
   <property name="text">
    <string>Risk Management Calculator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEditEntryPrice</tabstop>
  <tabstop>lineEditStopLoss</tabstop>
  <tabstop>lineEditLotSize</tabstop>
  <tabstop>lineEditTakeProfit</tabstop>
  <tabstop>lineEditCommission</tabstop>
  <tabstop>lineEditSpread</tabstop>
  <tabstop>lineEditSwap</tabstop>
  <tabstop>doubleSpinBoxRisk</tabstop>
  <tabstop>lineEditAccountBalance</tabstop>
  <tabstop>pushButtonCalculate</tabstop>
  <tabstop>lineEditRiskRewardRatio</tabstop>
  <tabstop>lineEditTotalCost</tabstop>
  <tabstop>lineEditBreakEvenPrice</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
